examples/experimental/export_dummy_example.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    session.experimental_enable_feature("otel_tracing")
</pre><p>Can you update the order here slightly to make it more clear?</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># start a trulens session
session = TruSession()

# (optional) if using another exporter to view traces, start the session with an otel exporter
session = TruSession(_experimental_otel_exporter=exporter)

# enable otel experimental feature: ## review note: even though this is done by default if the exporter is set above, it's clearer to have this laid out stepwise
session.experimental_enable_feature("otel_tracing")


</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Changed to use a property to set the exporter so I think this is fixed:

```python
# Create a TruLens session.
session = TruSession()

# Set the exporter.
session.experimental_otel_exporter = exporter

# (Optional) Enable otel_tracing. Note that this is not required if you set the
# exporter above. If you would like to trace using spans without an exporter,
# this step is required.
session.experimental_enable_feature("otel_tracing")
```